### ISSUE TITLE
Vaults: shard not saved on owner device (flag "keep one shard on my device" is on)

### ISSUE THREAD

**Issue by Alusar** [GuardianLabs/guardian-keyper#24]
steps:
1. create new vault, (flag "keep one shard on my device" is on)
2. add 2 guardians
3. split secret
4. as owner, open "shards" tab

actual result:
shard not showing up on owner device

expected result:
shard showing up on owner device
also, other guardians can scan his code to restore vault

**Comment by alexandrim0** [comment]
Shard is saved in fact.
We have no reason to see them because we can not manage them:
 - we have no flow to deliver ownership to other device
 - we not allow user to delete shards

**Comment by alexandrim0** [comment]
@ichorid do we need flow for delivering ownership to other device?

**Comment by ichorid** [comment]
Indeed, not seeing the owner's shard in the list is surprising. However, managing it in the same way as other shards could unnecessarily complicate the existing UX flows.
My suggestion is to still show the shard in the list of stored shards, but make it "inactive" and unmanageable (at least through the shards list). Something like this:
| Shards |
|--------|
| _Vault A (local)_ |
| Vault B |
| Vault C |

**Comment by kdiomidov** [comment]
@Alusar thanks for the heads up!

For now the issue seems not so critical to address it in the current version. I do agree that It might confuse users, but it doesn't keep anyone from restoring/accessing a "locally sharded" secret. We'll take care of it in the 3.x build.
